about.html

```diff
@@ -0,0 +1,160 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+      <script src="https://kit.fontawesome.com/57e3a5d1f4.js" crossorigin="anonymous"></script>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="assets/styles/css/main.css" />
+    <link rel="stylesheet" href="assets/styles/css/about.css" />
+    <link rel="stylesheet" href="assets/styles/css/reset.css" />
+    <link rel="stylesheet" href="assets/styles/css/hamburger-btn.css">
+    <title>Tehran International exhibition</title>
+  </head>
+  <body>
+    
+    
+    <nav class="navbar navbar-expand-sm d-none d-sm-block bg-dark navbar-dark justify-content-end">
+      <div class="container-fluid justify-content-end">
+      <ul class="navbar-nav ">
+        <li class="nav-item"><a href="" class="nav-link">
+          <i class="fa-brands fa-facebook-square"></i>
+        </a></li>
+        <li class="nav-item"><a href="" class="nav-link link-light">
+          <i class="fa-brands fa-twitter-square"></i>
+        </a></li>
+        <li class="nav-item"><a href="" class="nav-link link-light">Log out</a></li>
+        <li class="nav-item"><a href="" class="nav-link link-light">My Pages</a></li>
+        <li class="nav-item"><a href="" class="nav-link link-light">Persian</a></li>
+      </ul>
+    </div>
+    </nav>
+    <!--Mobile menu-->
+    <header class="top-nav">
+      <nav class="navbar navbar-expand-sm navbar-inverse navbar-light bg-light ">
+        <div class="container">
+         
+          <div class="navbar-header">
+            <a class="navbar-brand d-sm-block d-none ml-1 mar-3" href="index.html">
+              <img src="/assets/images/logo-sm.png" alt="small logo">
+            </a>
+          </div>
+          
+          <div class="container">
+            <nav class="navbar navbar-expand-lg navbar-light">
+              <div class="container-fluid">
+                <button class="navbar-toggler collapsed d-flex d-lg-none flex-column justify-content-around" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                  <span class="toggler-icon top-bar"></span>
+                  <span class="toggler-icon middle-bar"></span>
+                  <span class="toggler-icon bottom-bar"></span>
+                </button>
+                <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+                  <ul class="navbar-nav">
+                    <li class="nav-item px-4 active">
+                      <a  href="Index.html" class="nav-link">Home</a>
+                    </li>
```

Comment:
- [ ] Error is generated when this home page is clicked, Kindly fix it.

Reply:
Nothing is attached, Please give me the exact error, I don't have any errors!

Reply:
- It's because `Index.html` is starting with caps. It should start with lower case.

Reply:
Check again

Reply:
Html is case-sensitive??

Reply:
Would you send me "pseudo focus class " screenshot of the template, thank you so much

Reply:
Yes html is case sensitive

Reply:
I can't find this problem, please send me a screenshot of the template" Kindly add the pseudo focus class on the navigation bar link as seen in the Behance design attached below. This is the text surrounded with heavy red border"

Reply:
![Screenshot from 2022-02-12 13-56-50](https://user-images.githubusercontent.com/20245164/153708551-11c17900-f084-449a-a808-ac88def89375.png)

Reply:
👍🏽 The link has been fixed.
👍🏽 The pseudo focus class has been added.
- [ ] [OPTIONAL] The pseudo focus class on the nav-bar link as in the provided template has some padding that separates the text from the heavy red border. Kindly implement this.
#### Your Design
![1](https://user-images.githubusercontent.com/58692590/153712259-eae726fe-444f-46a3-920e-9001e3243973.png)
#### Figma Template
![11](https://user-images.githubusercontent.com/58692590/153712293-6aca16fd-6a3e-4326-bf67-f8da2bd07cf7.png)